dbt_project.yml

```
@@ -1,4 +1,22 @@
 name: 'fivetran_utils'
-version: '0.4.10'
+version: '0.5.0'
+profile: 'default'
 config-version: 2
-require-dbt-version: [">=1.3.0", "<2.0.0"]
+
+require-dbt-version: [">=1.10.6"]
```

@b-per maybe we still need a new version to remove the upper bound so that fusion works?


```suggestion
require-dbt-version: [">=1.3.0"]
```

---

Fusion doesn't look at those today (but might in the future)